crates/wasmtime/src/config.rs

```diff
@@ -109,6 +110,8 @@ pub struct Config {
     pub(crate) memory_init_cow: bool,
     pub(crate) memory_guaranteed_dense_image_size: u64,
     pub(crate) force_memory_init_memfd: bool,
+    #[cfg(feature = "async")]
```

Comment:
To avoid some extraneous `#[cfg]` below I think it's ok to remove this `#[cfg]` while still leaving the `#[cfg]` on the setter method